repairnator/src/main/java/fr/inria/spirals/repairnator/process/ProjectScanner.java

```diff
@@ -297,19 +294,20 @@ public String readWorkspaceFromInput(String input) throws IOException {
         return properties.getProperty("workspace");
     }
 
-    public List<Build> readBuildFromInput(String input) throws IOException {
-        List<Build> result = new ArrayList<Build>();
+    public List<BuildToBeInspected> readBuildFromInput(String input) throws IOException {
+        List<BuildToBeInspected> buildsToBeInspected = new ArrayList<BuildToBeInspected>();
 
         Properties properties = getPropertiesFromInput(input);
         String buildId = properties.getProperty("buildid");
         if (buildId != null) {
             Build build = BuildHelper.getBuildFromId(Integer.parseInt(buildId), null);
             if (build != null) {
-                result.add(build);
+                BuildToBeInspected buildToBeInspected = new BuildToBeInspected(build, build.getBuildStatus());
```

Comment:
ok for that case for now, you could also add a value "unknown" to the status of the build (see my previous comment concerning status of buildtobeinspected)